### PR TITLE
obs-packaging: Make distros file arch specific

### PR DESCRIPTION
**Pull request by nitkon**
Make distros file arch specific

Fixes: #535

Signed-off-by: Nitesh Konkar niteshkonkar@in.ibm.com

### CODE REVIEW

**Comment by nitkon** [comment]
@jcvenegas @jodh-intel 
/test

File: obs-packaging/create-repo-branch.sh [hunk 1]
```diff
@@ -47,7 +47,7 @@ read_repos(){
 		[[ "$p" != "#"* ]] || continue
 		repos+=("${p}")
 		echo "Adding distro: ${p}"
-	done < "${script_dir}/distros"
+	done < "${script_dir}/distros_${arch_target}"
```
Comment (devimc) [inline obs-packaging/create-repo-branch.sh]: I like the idea, but please rename the current `distros` file to `distros_x86_64`

**Comment by nitkon** [comment]
@devimc : Done. PTAL
/retest

**Comment by nitkon** [comment]
@jodh-intel PTAL

**Pull request merged**